src/main/java/com/mraof/minestuck/data/worldgen/MSConfiguredFeatureProvider.java

```diff
@@ -123,6 +127,12 @@ public static void register(BootstapContext<ConfiguredFeature<?, ?>> context)
 				new DiskConfiguration(RuleBasedBlockStateProvider.simple(Blocks.SLIME_BLOCK), BlockPredicate.matchesBlocks(Blocks.MYCELIUM, Blocks.DIRT), UniformInt.of(2, 6), 2)));
 		context.register(NETHERRACK_DISK, new ConfiguredFeature<>(MSFeatures.DISK.get(),
 				new DiskConfiguration(RuleBasedBlockStateProvider.simple(Blocks.NETHERRACK), BlockPredicate.alwaysTrue(), UniformInt.of(2, 3), 1)));
+		context.register(MAGMATIC_IGNEOUS_DISK, new ConfiguredFeature<>(MSFeatures.DISK.get(),
+				new DiskConfiguration(RuleBasedBlockStateProvider.simple(MSBlocks.MAGMATIC_IGNEOUS_STONE.get()), BlockPredicate.alwaysTrue(), UniformInt.of(2, 2), 1)));
+		context.register(BLACK_SAND_DISK, new ConfiguredFeature<>(MSFeatures.DISK.get(),
+				new DiskConfiguration(RuleBasedBlockStateProvider.simple(MSBlocks.BLACK_SAND.get()), BlockPredicate.alwaysTrue(), UniformInt.of(2, 4), 1)));
+		context.register(PUMICE_STONE_DISK, new ConfiguredFeature<>(MSFeatures.DISK.get(),
+				new DiskConfiguration(RuleBasedBlockStateProvider.simple(MSBlocks.PUMICE_STONE.get()), BlockPredicate.alwaysTrue(), UniformInt.of(2, 4), 1)));
```

Comment:
I would make these larger. You can reduce frequency with this change but I dont think they really bring much to the feel of the land at these sizes

Reply:
decided to give it a +1 increase, the others have been bumped up to 8 and black sand has also been made a bit more common for variety. pumice stands out too much so ill keep it at what it is rarity wise